examples/mnist-keras/README.md

```diff
@@ -52,4 +53,25 @@ bin/build.sh
 Finally, you can start the experiment from the "control" tab.
 
 ## Clean up
-You can clean up by running `docker-compose down`.
\ No newline at end of file
+You can clean up by running `docker-compose down`.
+
+## Connecting to a distributed deployment
+To start and remotely connect a client with the required dependencies for this example, start by downloading the `client.yaml` file. You can either navigate the reducer UI or run the following command.
+
+```bash
+curl -k https://<reducer-fqdn>:<reducer-port>/config/download > client.yaml
+```
+> **Note** make sure to replace `<reducer-fqdn>` and `<reducer-port>` with appropriate values.
+
+Now you are ready to start the client via Docker by running the following command.
+
+```bash
+docker run -d \
+  -v $PWD/client.yaml:/app/client.yaml \
+  -v $PWD/data:/var/data \
+  -e ENTRYPOINT_OPTS=--data_path=/var/data/mnist.npz \
+  --add-host=reducer:<reducer-ip> \
```

Comment:
reducer and combiner hostnames are specific to our docker-compose. If the user have deployed reducer and combiners separately without compose, these names might be different.

Reply:
Good point... what could be a good way to point this out in the docs? What determine these, DNS?